### DISCUSSION TITLE
Welcome, Abderrahim!

### DISCUSSION THREAD

**Discussion by AbdrrahimDahmani** [ParanoidUser/yolo#984] [Subscribe]
Hello I would like to contribute to this repository it sounds very fun

**Comment by ParanoidUser** [comment]
Ahoy, @AbdrrahimDahmani!

Thank you for your interest in this project! We're very excited to have you as a collaborator. Do you have any specific things in mind that you would like to work on? We're constantly looking for new and innovative ideas to share with community.

Here are just a few of the areas where help is most needed:

- We're always on the lookout for more exciting questions & puzzles to share with the community;
- Create, improve, and maintain badge earning guides and materials;
- Increase user engagement and attract new users interested in the project;
